rest/route/version_create.go

```diff
@@ -0,0 +1,50 @@
+package route
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/evergreen-ci/evergreen/model/user"
+	"github.com/evergreen-ci/evergreen/rest/data"
+	"github.com/evergreen-ci/evergreen/util"
+	"github.com/evergreen-ci/gimlet"
+)
+
+func makeVersionCreateHandler(sc data.Connector) gimlet.RouteHandler {
+	return &versionCreateHandler{sc: sc}
+}
+
+type versionCreateHandler struct {
+	ProjectID string          `json:"project_id"`
+	Message   string          `json:"message"`
+	Active    bool            `json:"activate"`
```

Comment:
Is this to allow users to create but not schedule versions?

Reply:
Correct